src/main/scala/org/globalforestwatch/layers/GrossEmissionsCo2OnlyCo2e.scala

```diff
@@ -2,13 +2,18 @@ package org.globalforestwatch.layers
 
 import org.globalforestwatch.grids.GridTile
 
-case class GrossEmissionsCo2OnlyCo2e(gridTile: GridTile,
-                                     model: String = "standard", kwargs: Map[String, Any])
+case class GrossEmissionsCo2OnlyCo2e(gridTile: GridTile,  model: String = "standard", kwargs: Map[String, Any])
   extends FloatLayer
     with OptionalFLayer {
 
-  val datasetName = "gfw_full_extent_co2_gross_emissions"
+//  val datasetName = "gfw_full_extent_co2_gross_emissions"
```

Comment:
This affects the TCL flagship analysis, is this different than the carbon data we released for TCL 2022? Do we need to update our maps and such on flagship with these layers? If that's the case, we'll upload to the API later anyway.

Reply:
I know. The tiles in gfw-files and gfw-data-lake should have the same values. As I mentioned elsewhere, I want to clean up the carbon layers so that there aren't duplicates and they all pull from the data-lake. I've switched the removals, emissions, and net flux back to the data-lake and will clean up other carbon layers in the next PR.